sw/device/tests/power_virus_systemtest.c

```diff
@@ -1273,9 +1284,18 @@ static void max_power(void) {
 
   // Request entropy during max power epoch. Since AES is so fast, realistically
   // we will only be able to request a single block of entropy.
+  csrng_wait_ready();
   mmio_region_write32(csrng.base_addr, CSRNG_CMD_REQ_REG_OFFSET,
                       csrng_instantiate_cmd_header);
 
+  uint32_t csrng_reseed_cmd = csrng_cmd_header_build(
+      kCsrngAppCmdReseed, kDifCsrngEntropySrcToggleEnable, /*cmd_len=*/0,
+      /*generate_len=*/0);
```

Comment:
For a moment I thought we maybe should build this command header outside of the max power epoch and just write it to CSRNG here. But then I noted that we trigger an instantiate with real entropy just before. So operations actually overlap which is nice.